WORKSPACE

```diff
@@ -17,14 +17,13 @@ load("@com_grail_bazel_toolchain//toolchain:rules.bzl", "llvm_toolchain")
 llvm_toolchain(
     name = "llvm_toolchain_12_0_0",
     absolute_paths = True,
-    llvm_mirror_prefixes = [
-        "https://sorbet-deps.s3-us-west-2.amazonaws.com/",
-        "https://artifactory-content.stripe.build/artifactory/github-archives/llvm/llvm-project/releases/download/llvmorg-",
-        "https://github.com/llvm/llvm-project/releases/download/llvmorg-",
-    ],
```

Comment:
Why is this being deleted?

This is required to get Sorbet to build on Stripe's internal network, and cannot be deleted.

Reply:
I had issues with it when prototyping, I forgot to revert it. Will add back in.